csharp/test/Ice/enums/Test.ice

```diff
@@ -167,6 +167,36 @@ enum FLSimpleEnum : byte
 }
 sequence<FLSimpleEnum> FLSimpleEnumSeq;
 
+[cs:attribute:System.Flags] unchecked enum MyFlags : uint
+{
+    E0 = 1,
```

Comment:
I think it would be best to pick one consistent for our literals instead of switching between decimal and hexadecimal like this. For flags I'd lean towards just keeping the hexadecimal for all of them, but maybe forgo the leading 0's where they're not necessary.